README.md

```diff
@@ -329,6 +329,10 @@ strings that they parse.
   (`major`, `premajor`, `minor`, `preminor`, `patch`, `prepatch`, or `prerelease`),
   or null if the versions are the same.
 
+### Comparators
+* `intersects(comparator)`: Return true if the comparators intersect
+* `satisfiesRange(range)`: Return true if the comparator intersects with any
```

Comment:
I think this is probably addressed by calling it `comparator.intersectsRange()`?  Or even cleaner, make `Comparator.intersects` take either a Range or Comparator as an argument.